recipes/zserio/all/conanfile.py

```diff
@@ -0,0 +1,96 @@
+import os
+import shutil
+
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.files import collect_libs, copy, download, get
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+
+required_conan_version = ">=1.54.0"
+
+class ZserioConanFile(ConanFile):
+    name = "zserio"
+    description = "Zserio C++ Runtime Library"
+    license = "BSD-3-Clause"
+    url = "https://github.com/conan-io/conan-center-index/"
+    homepage = "https://zserio.org"
+    topics = ("zserio", "cpp", "c++", "serialization")
+    package_type = "static-library"
+    settings = "os", "arch", "compiler", "build_type"
+
+    options = { "fPIC": [True, False] }
+    default_options = { "fPIC": False }
+
+    @property
+    def _min_cppstd(self):
+        return 11
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
+
+    def validate(self):
+        if self.settings.os not in ["Linux", "Windows"]:
+            raise ConanInvalidConfiguration("Zserio currently supports only Windows and Linux")
+
+        if self.settings.compiler.cppstd:
+            check_min_cppstd(self, self._min_cppstd)
+
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version]["runtime"],
+            pattern="runtime_libs/cpp/*", strip_root=True)
+        shutil.move("cpp/zserio", ".")
+        shutil.move("cpp/CMakeLists.txt", ".")
+        shutil.rmtree("cpp")
+
+        license_link = f"https://raw.githubusercontent.com/ndsev/zserio/v{self.version}/LICENSE"
+        download(self, license_link, "LICENSE")
```

Comment:
```suggestion
        download(self, filename="LICENSE", **source['license'])
```

Please, move it to the conandata.yml as well, so in case of any change in the future, we will know about just by the checksum. Use https://github.com/conan-io/conan-center-index/blob/master/recipes/npcap/all/conandata.yml#L12 as example.

Reply:
Done.